src/core.js

```diff
@@ -67,8 +68,8 @@ var QUnit,
 	objectValues = function( obj ) {
 		var key, val,
 			vals = QUnit.is( "array", obj ) ? [] : {};
-		for ( key in obj ) {
-			if ( hasOwn.call( obj, key ) ) {
+		for (key in obj) {
+			if (hasOwn.call( obj, key )) {
```

Comment:
Ok, I'll check it out. Everything passed the jshint and jscs tests, so I didn't notice these.

The actual changes this time are in the test.run and test.hook methods.